frontend/src/static/js/v_summary.js

```diff
@@ -41,9 +41,14 @@ window.vSummary = {
     return {
       filtered: [],
       filterSearch: '',
-      filterSort: 'displayName',
       filterSortReverse: false,
       filterGroupSelection: 'groupByRepos',
+      sortGroupSelection: 'searchPath',
+      sortingOption: '',
+      isSortingDsc: '',
+      sortWithinGroupSelection: 'name',
+      sortingWithinOption: '',
+      isSortingWithinDsc: '',
```

Comment:
minor nit, right now quite hard to know which is used in the code, which is use for UI

you can either group all those UI variables together, use the 'filter' keywork, add comments
choose any you think can fit, i think grouping and adding comments will be the best.

Reply:
Okay changed.